data_table.py

```diff
@@ -0,0 +1,31 @@
+import pandas as pd
+import numpy as np
+import dash
+from dash import dash_table, html
+
+df= pd.read_csv('csv/customers-1000.csv')
```

Comment:
I normally prefer to have hard coded strings like the file name in another variable just to know it can be change, for example:

```suggestion
CUSTOMERS_CSV_FILE_PATH = 'csv/customers-1000.csv'

df = pd.read_csv(CUSTOMERS_CSV_FILE_PATH)
```

For this case it's probably an overkill but on long codebases it helps to readability. A way to see it, it's like adding a name to these strings